kill.go

```diff
@@ -129,7 +130,27 @@ func kill(containerID, signal string, all bool) error {
 		return fmt.Errorf("Process not running inside container %s", containerID)
 	}
 
-	if err := vc.KillContainer(containerID, podStatus.ContainersStatus[0].ID, signals[signal]); err != nil {
+	signum, signalOk := signals[signal]
+	if !signalOk {
+		s, err := strconv.Atoi(signal)
+		if err != nil {
+			return fmt.Errorf("Failed to convert signal %s to int", signal)
+		}
+		signum = syscall.Signal(s)
+		// check whether signal is valid or not
+		for _, v := range signals {
+			if v == signum {
+				// signal is a valid signal
+				if err := vc.KillContainer(containerID, podStatus.ContainersStatus[0].ID, signum); err != nil {
```

Comment:
I'd prefer only a single call to `vc.KillContainer()` if possible.

Also, note #109 if you want to pick that up?